system-tests/README.md

```diff
@@ -178,7 +178,17 @@ Once completed, following services will start within their docker containers:
 
 (EDC Connectors will also be seeded with initial required data using a [postman collection](../deployment/data/MVD.postman_collection.json))
 
-_Note, the `Newman` docker container will automatically stop after seeding initial data from postman scripts and `cli-tools` container will also automatically stop after registering participants._
+> Note, the `Newman` docker container will automatically stop after seeding initial data from postman scripts.
+
+> The container `cli-tools` will turn into the state `healthy` after registering successfully all participants.
+
+Sample for confirming successful run of container `cli-tools`.
+
+```powershell
+PS C:\> docker ps
```

Comment:
```suggestion
docker ps -a
```

Otherwise if cli-tools fails, its entry is not shown. With `-a`:

```
25207e27673c   system-tests_cli-tools                    "/bin/sh -c \"/app/en…"   2 minutes ago   Exited (1) 2 minutes ago                                                                                           cli-tools
```